components/segmented/segmented.component.ts

```diff
@@ -122,10 +120,9 @@ export class NzSegmentedComponent implements OnChanges, ControlValueAccessor {
   private isDisabledFirstChange = true;
 
   protected value?: number | string;
-  protected animationState: null | { value: string; params: ThumbAnimationProps } = {
-    value: 'to',
-    params: thumbAnimationParamsOf()
-  };
+  protected readonly thumbStyle = signal<NgStyleInterface>(null!);
```

Comment:
question: why not defining the type of the signal with NgStyleInterface | null instead using null! ?